lib/shared/addon/components/form-key-value/component.js

```diff
@@ -122,8 +122,14 @@ export default Component.extend({
         return;
       }
 
-      var lines = str.split(/\r?\n/);
+      // if we paste an object with trailing commas the commas are not stripped
```

Comment:
Key-value fields are arbitrary values and might a password that ends in `,`

Reply:
okay cool, i wasn't sure about this, i'll remove it